app/controllers/products_controller.rb

```diff
@@ -0,0 +1,56 @@
+class ProductsController < ApplicationController
+  def index
+    @vendor_id = params[:vendor_id]
```

Comment:
Do `@vendor_id` and `@id` need to be instance variables? It seems like the view is only using `@products` so these two variables should be local variables